stubs/protobuf/google/protobuf/internal/well_known_types.pyi

```diff
@@ -63,29 +67,33 @@ class _FieldMaskTree:
     def AddLeafNodes(self, prefix: tAny, node: tAny) -> None: ...
     def MergeMessage(self, source: tAny, destination: tAny, replace_message: tAny, replace_repeated: tAny) -> None: ...
 
+_StructValue: TypeAlias = struct_pb2.Struct | struct_pb2.ListValue | str | float | bool | None
+_StructValueArg: TypeAlias = _StructValue | Mapping[str, _StructValueArg] | Sequence[_StructValueArg]
+
 class Struct:
-    def __getitem__(self, key: tAny): ...
-    def __contains__(self, item: tAny): ...
-    def __setitem__(self, key: tAny, value: tAny) -> None: ...
-    def __delitem__(self, key: tAny) -> None: ...
+    def __getitem__(self, key: str) -> _StructValue: ...
+    def __contains__(self, item: _StructValueArg) -> bool: ...
+    def __setitem__(self, key: str, value: _StructValueArg) -> None: ...
+    def __delitem__(self, key: str) -> None: ...
     def __len__(self) -> int: ...
-    def __iter__(self): ...
-    def keys(self): ...
-    def values(self): ...
-    def items(self): ...
-    def get_or_create_list(self, key: tAny): ...
-    def get_or_create_struct(self, key: tAny): ...
-    def update(self, dictionary: tAny) -> None: ...
+    def __iter__(self) -> Iterator[str]: ...
+    def keys(self) -> Iterator[str]: ...
+    def values(self) -> Iterator[_StructValue]: ...
+    def items(self) -> Iterator[tuple[str, _StructValue]]: ...
```

Comment:
These both return lists: https://github.com/protocolbuffers/protobuf/blob/ca561b0ef5ea5a9ca908b002e574e20e8b8fb1b8/python/google/protobuf/internal/well_known_types.py#L500-L504

```suggestion
    def values(self) -> list[_StructValue]: ...
    def items(self) -> list[tuple[str, _StructValue]]: ...
```

Reply:
Hmm, well `Iterator` is incorrect either way, since a `list` is not an `Iterator` (lists don't have a `__next__` method). I won't object if you want to change it to `Iterable` instead of `list` 🙂